### DISCUSSION TITLE
Downloading maps - shows progress on another maps

### DISCUSSION THREAD

**Discussion by airald** [devemux86/cruiser#374] [Q&A] [answered]
I've downloaded many maps on my backup phone. Then it shows progress on many other maps. Just progress and speed, not the downloading icon on the left.

I repeated this test on another phone but just downloading one map. The result is the same.

Download: France (Mapsforge)
Then progress is shown on:
Freizeitkarte:
Andorra (that's also downloaded in the phone) 35 kb/s 0%
Cyprus 12.7 MB/s 24%
Iceland 12.7 MB/s 24%
Montenegro 12.7 MB/s 24%
Ukraine 12.7 MB/s 24%

OpenAndroMaps
Voluntary MF5 (also downloaded) 12.7 MB/s 24%

Germany folder and also some items inside.
It seems that 7th or 8th position is always "downloading", if I enter inside Germany folder, the 7th position is also showing progress.

Progress and speed does not changes. If I exit map menu and enter again, all is OK. Downloaded maps are just maps I selected.
I repeated the test and progress is shown on the same countries except Ukraine.

The 2 phones are Xiaomi running MIUI 14.

**Comment by devemux86** [comment]
@airald thanks for the report.

What download manager have you selected, "Android" or "Internal"?

**Reply by airald** [reply]
Android

**Comment by devemux86** [comment]
I can reproduce it and will be fixed in the next Cruiser update.

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.35](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
